package.json

```diff
@@ -8,14 +8,18 @@
     "@testing-library/user-event": "^13.5.0",
     "antd": "^5.22.1",
     "axios": "^1.5.0",
+    "base-64": "^1.0.0",
     "chart.js": "^4.4.7",
+    "convert-array-to-csv": "^2.0.0",
     "date-fns": "^4.1.0",
+    "dayjs": "^1.11.13",
     "jwt-decode": "^3.1.2",
     "moment": "^2.29.4",
     "react": "^18.2.0",
     "react-bar-loader": "^1.0.0",
     "react-chartjs-2": "^5.3.0",
     "react-dom": "^18.2.0",
+    "react-highlight-words": "^0.21.0",
```

Comment:
Same here, I will suggest you look at all the packages installed and see if you are using them, if not in use please remove them